agent/lib_aws_sdk_php.c

```diff
@@ -14,10 +14,358 @@
+below:
+//clang-format off
+$result = $client->receiveMessage(array(
+    // QueueUrl is required
+    'QueueUrl' => 'string',
+    'AttributeNames' => array('string', ... ),
+    'MessageAttributeNames' => array('string', ... ),
+    'MaxNumberOfMessages' => integer,
+    'VisibilityTimeout' => integer,
+    'WaitTimeSeconds' => integer,
+));
+
+$result = $client->sendMessage(array(
+    // QueueUrl is required
+    'QueueUrl' => 'string',
+    // MessageBody is required
+    'MessageBody' => 'string',
+    'DelaySeconds' => integer,
+    'MessageAttributes' => array(
+        // Associative array of custom 'String' key names
+        'String' => array(
+            'StringValue' => 'string',
+            'BinaryValue' => 'string',
+            'StringListValues' => array('string', ... ),
+            'BinaryListValues' => array('string', ... ),
+            // DataType is required
+            'DataType' => 'string',
+        ),
+        // ... repeated
+    ),
+));
+
+
+$result = $client->sendMessageBatch(array(
+    // QueueUrl is required
+    'QueueUrl' => 'string',
+    // Entries is required
+    'Entries' => array(
+        array(
+            // Id is required
+            'Id' => 'string',
+            // MessageBody is required
+            'MessageBody' => 'string',
+            'DelaySeconds' => integer,
+            'MessageAttributes' => array(
+                // Associative array of custom 'String' key names
+                'String' => array(
+                    'StringValue' => 'string',
+                    'BinaryValue' => 'string',
+                    'StringListValues' => array('string', ... ),
+                    'BinaryListValues' => array('string', ... ),
+                    // DataType is required
+                    'DataType' => 'string',
+                ),
+                // ... repeated
+            ),
+        ),
+        // ... repeated
+    ),
+));
+
+//clang-format on
+*/
+
+char* nr_lib_aws_sdk_php_get_command_name(NR_EXECUTE_PROTO) {
+  zval* command_name = NULL;
+  char* command_name_string = NULL;
+
+  /* get the command_name. */
+  command_name = nr_php_arg_get(1, NR_EXECUTE_ORIG_ARGS);
+
+  if (0 != nr_php_is_zval_non_empty_string(command_name)) {
+    command_name_string = nr_strdup(Z_STRVAL_P(command_name));
+  }
+
+  nr_php_arg_release(&command_name);
+  return command_name_string;
+}
+
+void nr_lib_aws_sdk_php_sqs_handle(nr_segment_t* segment, NR_EXECUTE_PROTO) {
+  char* command_arg_value = NULL;
+  char* command_name_string = NULL;
+  bool instrumented = false;
+  nr_segment_message_params_t message_params = {
+      .library = "SQS",
+      .destination_type = NR_MESSAGE_DESTINATION_TYPE_QUEUE,
+      .messaging_system = "aws_sqs",
+  };
+  nr_segment_cloud_attrs_t cloud_attrs = {0};
+
+  if (NULL == segment) {
+    return;
+  }
+
+  command_name_string
+      = nr_lib_aws_sdk_php_get_command_name(NR_EXECUTE_ORIG_ARGS);
+
```

Comment:
Check if `command_name_string` is `NULL`, and return early if it is.

Reply:
Location has changed, but we do check for null here:
https://github.com/newrelic/newrelic-php-agent/pull/1003/commits/c1f163f7c43f1b731e1b94a515b0ec47cf4383f5